staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -23,10 +23,16 @@ import (
 	"github.com/spf13/pflag"
 	"k8s.io/apiserver/pkg/admission"
 	"k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle"
 	"k8s.io/apiserver/pkg/server"
 	"k8s.io/client-go/kubernetes"
 )
 
+var (
+	// defaultPlugiNames a list that holds default plugin names that we want to apply by default, order matters.
```

Comment:
defaultPluginNames

Reply:
I definitely need to invest in a spell checker :)